crates/solidity/testing/solc/src/keywords/mod.rs

```diff
@@ -225,6 +225,8 @@ impl TestCase {
                 "State mutability can only be specified for address types.",
                 "Cannot use instruction names for identifier names.",
                 "Cannot use builtin function name",
+                "Cannot use instruction name",
```

Comment:
nit: maybe removing the older `Cannot use instruction names for identifier names`, as the new one will always match it?